### PR TITLE
[INLONG-6861][Dashboard] Support extract node of Apache Hudi

### LINKED ISSUE #6861: [Feature][Dashboard] Support Apach Hudi source management
### Description

[Feature][Dashboard] Support Apach Hudi source management

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ZuoFengZhang**
### Prepare a Pull Request
 
- [INLONG-6861][Dashboard] Support extract node of Apache Hudi

- Fixes #6861 

### Motivation

企业微信截图_f7c3cdd5-a6f4-4d26-bf79-eccd3c016137

企业微信截图_6ed026ba-40b2-489d-8164-a46f8228e544

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-dashboard/src/locales/cn.json [hunk 1]
```diff
@@ -157,6 +164,10 @@
   "meta.Sinks.ES.FlushRecordUnit": "条",
   "meta.Sinks.ES.RetryTimes": "重试次数",
   "meta.Sinks.ES.RetryTimesUnit": "次",
+  "meta.Sinks.ES.EsVersion": "Elasticsearch 版本",
```
Comment (leezng) [inline inlong-dashboard/src/locales/cn.json:167]: Is the irrelevant code here because there is no rebase? You need to remove this part of the code.

**Review by leezng — CHANGES_REQUESTED** [review]

File: inlong-dashboard/src/locales/cn.json [hunk 2]
```diff
@@ -157,6 +164,9 @@
   "meta.Sinks.ES.FlushRecordUnit": "条",
   "meta.Sinks.ES.RetryTimes": "重试次数",
   "meta.Sinks.ES.RetryTimesUnit": "次",
+  "meta.Sinks.ES.DocumentType": "文档类型",
+  "meta.Sinks.ES.PrimaryKey": "主键",
+  "meta.Sinks.ES.PrimaryKeyHelp": "是否将某字段作为 `_id` 字段，如果不指定，则 ES 将自动为每个文档生成唯一的 `_id`",
   "meta.Sinks.ES.FieldName": "字段名",
```
Comment (healchow) [inline inlong-dashboard/src/locales/cn.json:170]: Please rebase from the master branch, again.

Comment (ZuoFengZhang) [reply inlong-dashboard/src/locales/cn.json:170]: ok

Comment (ZuoFengZhang) [reply inlong-dashboard/src/locales/cn.json]: done

**Review by healchow — APPROVED** [review]
LGTM

**Review by leezng — APPROVED** [review]

**Pull request merged**